.github/workflows/wheels-recipe.yaml

```diff
@@ -253,3 +253,26 @@ jobs:
         with:
           name: wheels-windows-${{ matrix.cibw_arch }}-${{ strategy.job-index }}
           path: ./dist/*.whl
+
+  build_pyodide_wheel:
+    name: Build python cp312-* pyodide_wasm32 wheels on ubuntu-latest
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
```

Comment:
Fetch depth required?

Reply:
I'm not sure if it is, but I saw other jobs in this file using it, so I decided to keep it.

Reply:
Check if you need it: https://github.com/actions/checkout

Reply:
AFAICS, we aren't using any tool that requires anything but a shallow clone; will drop it.

Reply:
Addressed via 1ba76682cdd233c56fa4a5a8c2884ad2ce0fe712